libs/generic-view/store/src/lib/hooks/use-locked-device-handler.ts

```diff
@@ -17,9 +17,9 @@ import { getAPIConfig } from "../get-api-config"
 export const useLockedDeviceHandler = () => {
   const dispatch = useDispatch<Dispatch>()
   const deviceId = useSelector(selectActiveApiDeviceId)
-  const deviceLocked = useSelector((state: ReduxRootState) => {
-    return selectApiError(state, ApiError.DeviceLocked)
-  })
+  const deviceLocked = useSelector((state: ReduxRootState) =>
+    deviceId ? selectApiError(state, deviceId, ApiError.DeviceLocked) : false
+  )
```

Comment:
🟢 

I noticed that the deviceLocked value is already used at least twice in this PR. Perhaps it's worth considering creating a dedicated selector to handle this? This would allow for centralized management of the logic and help avoid code duplication.